code/src/index.css

```diff
@@ -5,9 +5,241 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #222;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main-container {
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+/*HEADER*/
+.header-container {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin: 20px 20px 0 20px;
+  flex-wrap: wrap;
+}
+
+.header-title {
+  width: 100%;
+  color: white;
+  font-size: 20px;
+}
+
+.header-line { 
+  width: 100%;
+  border-top: 2px solid #3f3f3f;  
+}
+
+/*ALBUM CONTAINER*/
+.album-container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+  margin-top: 20px;
+}
+ 
+ .album-card {
+    width: 89%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    font-size: 14px;
+ }
+
+ .album-name{
+  font-weight: 800;
+  font-size: 14px;
+  color: #ffff;
+  text-decoration: none;
+  padding-top: 10px;
+ }
+
+ .album-name:hover{
+  text-decoration: underline;
+ }
+ 
+ .artists > a:link {
+   display: inline-flex;
+   font-size: 14px;
+   color:  #a0a0a0;
+   text-decoration: none;
+   padding-top: 5px;
+   
+ }
+ 
+ .artists a:visited {
+  color:  #a0a0a0;
+  text-decoration: none;
+} 
+
+.artists > a:hover {
+  color: #ffff;
+  border-bottom:1px solid rgb(255, 255, 255);
+}
+
+
+
+.artists > :not(:last-child):not(:first-child):before {
+  content: ", ";
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+.artists > :last-child:before {
+  content: " & ";
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+.artists > :only-child:before {
+  content: "";
+}
+ .album-image {
+   width: 100%;
+   object-fit: contain;
+ }
+
+/*IMAGE CONTAINER */
+.image-container {
+  width: 100%;
+  position: relative;
+}
+
+.image-overlay {
+  top: 0;
+  z-index: 1;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.image-container:hover .image-overlay {
+  background: rgba(0, 0, 0, 0.6);
+  transition: 0.3s ease;
+}
+
+.image-overlay .play-icon{
+ transform: scale(1.8);
+ transition: 0.3s ease;
+}
+
+
+.icons-container {
+  position: absolute;
+  width: 100%;
+  display: none;
+  justify-content: center;
+  align-items: center;
+  top: 50%;
+}
+
+.image-container:hover .icons-container {
+  display: flex;
+}
+
+/*ICONS*/
+.icons-container img {
+  height: 35px;
+  margin: 0 15px;
+  opacity: 50%;
+}
+
+.heart-icon:hover, .dots-icon:hover {
+  transform: scale(1.2);
+  transition: 0.3s ease;
+  opacity: 100%;
+}
+
+
+.play-icon:hover {
+  transform: scale(1.8);
```

Comment:
If you change the scale to ex 2 the play icon will also transform. Now it is the same scale in row 135 and 168.